prompt2model/utils/config.py

```diff
@@ -0,0 +1,14 @@
+"""Place to store all the default configurations."""
+MAX_SUPPORTED_BATCH_SIZE = 4
+
+DEFAULT_HYPERPARAMETERS_SPACE = {
+    "min_num_train_epochs": 10,
+    "max_num_train_epochs": 20,
+    "save_strategy": ["epoch", "steps", "no"],
+    "evaluation_strategy": ["epoch", "no"],
+    "per_device_train_batch_size": [2, 4],
```

Comment:
I feel like `per_device_train_batch_size` is not a good hyperparameter to tune, because you should generally just use the maximum batch size that fits in your machine's memory.
```suggestion
    "per_device_train_batch_size": 2,
```

Reply:
Let's discuss on [this](https://github.com/neulab/prompt2model/pull/315#discussion_r1364499905), and after that will commit the changes.